grace/server.go

```diff
@@ -120,7 +123,7 @@ func (srv *Server) ListenAndServeTLS(certFile, keyFile string) (err error) {
 			log.Println(err)
 			return err
 		}
-		err = process.Kill()
+		err = process.Signal(syscall.SIGINT)
```

Comment:
But in UNIX like OS Kill will force to kill the process, it won't wait for the parent gracefully exit, if INT is not support on windows then we can change it to another one.